tests/Feature/TestCase.php

```diff
@@ -16,11 +16,55 @@ protected function setUp(): void
     {
         parent::setUp();
 
+        $this->loadMigrationsFrom(__DIR__.'/Support/Database/migrations');
+        $this->artisan('migrate', ['--database' => 'testing'])->run();
```

Comment:
I'm confused about this because migrations were going great on my side, you might not have PHP UNIT configured nicely in your local environment